src/animate/load.js

```diff
@@ -73,6 +76,11 @@ const load = function(options, parent, complete, basePath) {
         complete: null
     }, options || {});
 
+    const Resource = PIXI.loaders.Resource;
+    Resource.setExtensionLoadType('wav', Resource.LOAD_TYPE.AUDIO);
+    Resource.setExtensionLoadType('mp3', Resource.LOAD_TYPE.AUDIO);
+    Resource.setExtensionLoadType('ogg', Resource.LOAD_TYPE.AUDIO);
```

Comment:
This will get called every-time `PIXI.animate.load` is called. Could we push this out of the load function?